build-bin/docker/docker_arch

```diff
@@ -35,9 +35,6 @@ case ${docker_arch} in
   aarch64* )
     docker_arch=arm64
     ;;
-  s390x* )
```

Comment:
Something I don't get in this PR is that we are "adding support for s390x" but in the other hand we don't have a case in this switch meaning that we will go to `"Unsupported DOCKER_ARCH: ${docker_arch}"`, is that correct?